Essentials/src/main/java/com/earth2me/essentials/UserData.java

```diff
@@ -322,7 +323,8 @@ public void setJail(final String jail) {
         final List<String> list = new ArrayList<>();
         if (getMailAmount() != 0) {
             for (MailMessage mail : getMailMessages()) {
-                list.add(mail.getMessage());
+                // I hate this code btw
+                list.add(mail.isLegacy() ? mail.getMessage() : ChatColor.GOLD + "[" + ChatColor.RESET + mail.getSenderUsername() + ChatColor.GOLD + "] " + ChatColor.RESET + mail.getMessage());
```

Comment:
...after my previous review I realised this would've returned a translated string in the past.

Not sure if it's worth doing the `mailFormat` translation, keeping this hardcoded as it is now, or undoing this change entirely

Yes I'm indecisive shh

Reply:
hardcoded, people need to move off this api